packages/@aws-cdk/aws-stepfunctions-tasks/lib/dynamodb/shared-types.ts

```diff
@@ -108,82 +110,131 @@ export class DynamoProjectionExpression {
 }
 
 /**
- * Class to generate AttributeValue
+ * Map of string to AttributeValue
+ */
+export interface DynamoAttributeValueMap {
+  [key: string]: DynamoAttributeValue;
+}
```

Comment:
This isn't a valid thing to do (`jsii` will currently silently ignore the `[key: string]: DynamoAttributeValue` declaration altogether).

Reply:
good catch, i'm just going to get rid of the interface for now.

I tried changing this to a type but `jsii` also throws a warning about that as well indicating it'll ignore the `TypeAliasDefinition`